src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrentSettings.cs

```diff
@@ -73,6 +73,9 @@ public QBittorrentSettings()
         [FieldDefinition(13, Label = "DownloadClientQbittorrentSettingsContentLayout", Type = FieldType.Select, SelectOptions = typeof(QBittorrentContentLayout), HelpText = "DownloadClientQbittorrentSettingsContentLayoutHelpText")]
         public int ContentLayout { get; set; }
 
+        [FieldDefinition(14, Label = "DownloadClientQbittorrentSettingsPropagateTags", Type = FieldType.Checkbox, HelpText = "DownloadClientQbittorrentSettingsPropagateTagsHelpText")]
+        public bool PropagateTags { get; set; }
```

Comment:
To match the suggested naming.

```suggestion
        public bool AddSeriesTags { get; set; }
```